tests/modules/dastool/dastool/test.yml

```diff
@@ -0,0 +1,29 @@
+- name: dastool dastool test_dastool_dastool
+  command: nextflow run tests/modules/dastool/dastool -entry test_dastool_dastool -c tests/config/nextflow.config
+  tags:
+    - dastool/dastool
+    - dastool
+  files:
+    - path: output/dastool/test.scaffolds2bin.tsv
+      md5sum: 6e46c0be14dded7cb13af38f54feea47
+    - path: output/dastool/test.seqlength
+      md5sum: b815a5811008c36808a59b1d0dcfab24
+    - path: output/dastool/test_DASTool.log
+      contains: 
+        - 'DAS Tool run on'
+    - path: output/dastool/test_DASTool_scaffolds2bin.txt
+      md5sum: 6e46c0be14dded7cb13af38f54feea47
+    - path: output/dastool/test_DASTool_summary.txt
+      md5sum: a3efa8717b30dfada78dc5ae9a3dc396
+    - path: output/dastool/test_proteins.faa.archaea.scg
+      md5sum: e79d82eecee25821d1658ea4f082601d
+    - path: output/dastool/test_proteins.faa.bacteria.scg
+      md5sum: 8132cfb17cf398d41c036ead55c96ffe
+    - path: output/dastool/test_test.scaffolds2bin.tsv.eval
+      md5sum: a3efa8717b30dfada78dc5ae9a3dc396
+    - path: output/metabat2/bins/test.1.fa.gz
+      md5sum: 2b297bf557cc3831b800348859331268
+    - path: output/metabat2/test.tsv.gz
+      md5sum: 619338fa5019e361d5545ce385a6961f
```

Comment:
```suggestion
```

Reply:
It is reproducible with Docker/Singularity, so I would rather keep the md5sum check, and let it fail for conda.